examples/space.html

```diff
@@ -29,6 +30,8 @@
     <vr-cube z="-300" width="50" height="50" depth="50" y="25" x="100"></vr-cube>
     <vr-cube z="-300" width="50" height="50" depth="50" y="25" x="-100"></vr-cube>
     <vr-grid size="1000"></vr-grid>
+    <!--<vr-video z="-500" y="100" width="88" height="50" src="https://raw.githubusercontent.com/Mozilla-NTU/slidesMay2015/gh-pages/media/bunny.mp4" loop></vr-video>-->
+    <vr-audio z="-450" y="50" src="cricket.mp3" autoplay loop></vr-audio>
```

Comment:
It would be cool to have a visual reference of the sound position. What about making it a child of one of the cubes? So as the user moves around you can see where the sound is coming from.